.github/workflows/benchmarks.yml

```diff
@@ -6,19 +6,34 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
-          java-version: 11
+          distribution: 'adopt'
+          java-version: 15
+      - uses: actions/cache@v2
```

Comment:
Curious whether instead of caching the CI run, we could run an RSocket https://hub.docker.com/r/gradle/build-cache-node/ and use for speeding up local development.  Will sync with you in slack.